src/load.ts

```diff
@@ -78,6 +78,54 @@ export async function load(
     }
 }
 
+/**
+ * Loads the data from a CDN and returns an instance of AzureAppConfiguration.
+ * @param endpoint  The URL to the CDN.
+ * @param options  Optional parameters.
+ */
+export async function loadCdn(endpoint: URL | string, options?: AzureAppConfigurationOptions): Promise<AzureAppConfiguration>;
+
+export async function loadCdn(
+    endpoint: string | URL,
+    appConfigOptions?: AzureAppConfigurationOptions
+): Promise<AzureAppConfiguration> {
+    const startTimestamp = Date.now();
+    if (typeof endpoint === "string") {
+        try {
+            endpoint = new URL(endpoint);
+        } catch (error) {
+            if (error.code === "ERR_INVALID_URL") {
+                throw new Error("Invalid endpoint URL.", { cause: error });
+            } else {
+                throw error;
+            }
+        }
+    }
+    const emptyTokenCredential: TokenCredential = {

```

Comment:
I think we can just call `load(endpoint, emptyTokenCredential, appConfigOptions)` without any extra validations here. `load` takes care of everything else.